dataStorage/dataStorage.py

```diff
@@ -10,7 +12,14 @@ def store(location, pickleable, use_relative_location=True):
     :param use_relative_location: bool
     :return: An event containing a signal that describes success or failure
     """
-    _store_json(settings.get_storage_directory() + location, pickleable)
+    return _store_json(get_storage_directory() + location, pickleable)
+
+
+# TODO: need test
+def store_image(location, img):
```

Comment:
Needs unit tests, documentation, and perhaps some refactoring? For example, does not check if image already exists or if valid img path.